plugins/tfidf.cc

```diff
@@ -238,15 +241,39 @@ run(const ProcedureRunConfig & run,
         auto output = createDataset(server, outputDataset, onProgress, true /*overwrite*/);
 
         Date applyDate = Date::now();
+        ColumnName columnName("count");
 
-        std::vector<std::tuple<ColumnName, CellValue, Date> > row;
-        row.reserve(dfs.size());
-
-        for (auto& df : dfs) {
-            row.emplace_back(ColumnName(df.first), df.second/*(float)count*/, applyDate);
+#if 0
```

Comment:
I will.  It may be put back as early as this week but, as pointed out, this is already in github!